database/src/main/kotlin/no/nav/su/se/bakover/database/avstemming/AvstemmingPostgresRepo.kt

```diff
@@ -76,9 +76,9 @@ internal class AvstemmingPostgresRepo(
         tilOgMed: Tidspunkt
     ): List<Utbetaling.OversendtUtbetaling> =
         dataSource.withSession { session ->
-            val fraOgMedCondition = """(avstemmingsnøkkel ->> 'opprettet')::timestamptz >= :fom"""
-            val tilOgMedCondition = """(avstemmingsnøkkel ->> 'opprettet')::timestamptz <= :tom"""
-            """select * from utbetaling where $fraOgMedCondition and $tilOgMedCondition"""
+            val fraOgMedCondition = """(u.avstemmingsnøkkel ->> 'opprettet')::timestamptz >= :fom"""
+            val tilOgMedCondition = """(u.avstemmingsnøkkel ->> 'opprettet')::timestamptz <= :tom"""
+            """select u.*, s.saksnummer from utbetaling u inner join sak s on s.id = u.sakId where $fraOgMedCondition and $tilOgMedCondition"""
```

Comment:
Er dette årsaken til at vi tar vare på begge?

Reply:
Vi trenger ikke gjøre select på sakId for å gjøre joinen, det er bare mer praktisk, siden vi trenger den ved insert og det føles unødvendig å gjøre den nullable. Vi må uansett hente utbetalinger fra sak for å regne oss frem til `refDelytelseId`. Det gjør vi nå ved å hente saken i sin helhet i UtbetalingService, men her kunne vi gjort optimaliseringer.